File: C++/B/61.cpp
#include <iostream>
#include <vector>
using namespace std;
int main()
{
    int n, m, arr[50] = {0};
    cin >> n >> m;
    vector<int> a(m), b(m);
    for (int i = 0; i < m; i++)
    {
        cin >> a[i] >> b[i];
        arr[a[i] - 1]++;
        arr[b[i] - 1]++;
    }
    for (int i = 0; i < n; i++)
    {
        cout << arr[i] << endl;
    }
}

File: ruby/B/61.rb
n, m = gets.split.map(&:to_i)
arr = Array.new(50, 0)
m.times do
  a, b = gets.split.map(&:to_i)
  arr[a - 1] += 1
  arr[b - 1] += 1
end
for i in 0..n - 1
  p arr[i]
end
